### PR TITLE
Rename to bevy_new_2d

### PR DESCRIPTION
**Pull request by benfrankel**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -35,4 +35,4 @@
     </script>
 </body>
 
-</html>
+</html>
```
Comment (janhohenheim) [inline web/index.html:38]:
```suggestion
</html>

```

**Review by janhohenheim — APPROVED** [review]

**Comment by askor** [comment]
Nobody asked, but I feel this is the just the best template for bevy overall, so naming it "2D" seems limiting 😅

**Comment by Sapein** [comment]
I mean this template is, generally, focused on 2D atm.

**Comment by benfrankel** [comment]
> Nobody asked, but I feel this is the just the best template for bevy overall, so naming it "2D" seems limiting 😅

That's true today, but this rename is moreso preparing for a future where this template is more specialized for 2D and there are other templates for 3D, RPG, FPS, etc.

**Pull request merged**